dts/common/yaml/uart.yaml

```diff
@@ -22,4 +22,9 @@ properties:
       category: required
       description: Clock gate information
       generation: define
+  - label:
+      type: string
+      category: optional
+      description: Human readable string describing the device
```

Comment:
Best comment on this on PR #208.  However I'm leaving the description vague as its the Zephyr usage of the label that is for API identifier of the device.